src/patternfly/components/Tabs/tabs.scss

```
@@ -156,6 +156,18 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
   --pf-c-tabs__add--c-button--PaddingBottom: var(--pf-c-tabs__link--PaddingBottom);
   --pf-c-tabs__add--c-button--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
 
+  // Overflow menu toggle icon
+  --pf-c-tabs__link-toggle-icon--Color: var(--pf-global--Color--200);
+  --pf-c-tabs__link-toggle-icon--Transition: .2s ease-in 0s;
+  --pf-c-tabs__link-toggle-icon--Rotate: 0;
+  --pf-c-tabs__link-toggle-icon--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-tabs__link-toggle-icon--PaddingTop: 3px;
+  --pf-c-tabs__link--m-expanded__toggle-icon--Color: var(--pf-global--Color--100);
+  --pf-c-tabs__link--m-expanded__toggle-icon--Rotate: 90deg;
+  --pf-c-tabs__item--hover--Color: var(---pf-global--Color--100);
```

typo here - but is there a reason this hover state is on the `__item` and not the `__link`? Typically when we change a style when you're hovered/focused/active on something interactive - or otherwise interacting with the interactive thing, that the style change is visual feedback for) - we try to put the style on that thing if possible so as not to accidentally trigger a style change when clicking or whatever won't trigger a change on the interactive thing.

```suggestion
  --pf-c-tabs__item--hover--Color: var(--pf-global--Color--100);
```

---

No reason other than me being unsure which things should be on the `__link` vs the `__item`. Will update.